release-notes/opendistro-for-elasticsearch-knn.release-notes-1.12.0.0.md

```diff
@@ -0,0 +1,17 @@
+## Version 1.12.0.0 Release Notes
+
+Compatible with Elasticsearch 7.10.0
+
+### Features
+
+* Support for hamming bit distance in custom scoring ([#267](https://github.com/opendistro-for-elasticsearch/k-NN/pull/267))
+* k-NN plugin support for Elasticsearch version 7.10.0 ([#271](https://github.com/opendistro-for-elasticsearch/k-NN/pull/271))
+
+### Enhancements
+
+* Bump odfe version to 1.12 ([#273](https://github.com/opendistro-for-elasticsearch/k-NN/pull/273))
+
+### Infrastructure
```

Comment:
Could we just manually remove this section? It is a little bit confusing, because we made the change and then reverted it. I think it would be better to just not have it.

Reply:
i agree.

Reply:
Was under impression to have all the Prs going in the release to be documented. But makes sense to remove this to avoid confusion.